src/Mailer/Email.php

```diff
@@ -2084,10 +2085,20 @@ protected function _logDelivery($contents)
         }
         Log::write(
             $config['level'],
-            PHP_EOL . $contents['headers'] . PHP_EOL . PHP_EOL . $contents['message'],
+            PHP_EOL . $this->_isKeyArray($contents['headers']) . PHP_EOL . PHP_EOL . $this->_isKeyArray($contents['message']),
             $config['scope']
         );
     }
+    
+    
+    /**
+     * @param null $key
+     * @return null|string
```

Comment:
If you cast to string in your ternary we can just use string here:

    ... : $key

Reply:
good point it makes sense to ensure that it will return a string